commands/init.js

```diff
@@ -5,22 +5,24 @@ const Util = require('../utils/Util');
 const { SOURCE_MAP } = require('../utils/constants');
 const { NoSelectionElementsError } = require('../utils/errors');
 const { ChannelInCombat, CombatNotFound } = require('../yearzero/YZCombat');
+const { __ } = require('../lang/locales');
 
 const YargsParser = require('yargs-parser');
 const YARGS_PARSE_COMBATANT = {
-	array: ['p', 'note', 'name', 'group', 'controller'],
-	boolean: ['h'],
-	number: ['n'],
-	string: ['p', 'hp', 'max', 'ar', 'speed', 'haste'],
+	array: ['place', 'note', 'name', 'group', 'controller'],
```

Comment:
Did you triple-checked all your changes here?

Reply:
I did check it and the parser puts the aliases as separate properties, so it wouldn't matter.
But i will do another few tests later, just to make sure.

Reply:
I went through all instances where the arguments are used and checked them again and found no usage of the old notation.
I also played through a few init actions without any errors.